seed/client/src/main/resources/application.conf

```diff
@@ -1,4 +1,3 @@
 name = "SeedClient"
 host = "localhost"
-port = 19683
-port = ${SEED_PORT}
\ No newline at end of file
+port = 19683
```

Comment:
Sorry, didn't realise you'd fixed this over here in this PR. I fixed it a slightly different way in #13

Reply:
oh yeah the optional env var is probably cleaner; I never saw use using it anywhere but it doesn't hurt to keep it as an option